src/main/java/seedu/address/model/itinerary/ItineraryAttraction.java

```diff
@@ -47,7 +49,7 @@ public Name getName() {
      * Returns the attraction in this itinerary attraction.
      */
     public Attraction getAttraction() {
-        return this;
+        return attraction;
```

Comment:
return super; // if you want the Attraction
return this; // if you want ItineraryAttraction

Reply:
wait i want the attraction
but it was return this originally

Reply:
ok nvm i got it haha

Reply:
did return super work?

Reply:
wait.. why you change back to return this? you shld use super if you need only Attraction